spec.md

```diff
@@ -99,7 +97,8 @@ The `Span` interface must have the following capabilities:
 
 The `Tracer` interface must have the following capabilities:
 
-- Start a `Span` that has no parent, commonly referred to as a *root* `Span` **(py: `start_trace`, go: `StartTrace`)**
+- Start a new child `Span` with a given operation name. With no parent, commonly referred to as a *root* `Span`, no trace attributes exist. With a parent any trace attributes must be passed through into the created child. **(py: `create_span`, go: `CreateSpan`)**
```

Comment:
I think this may not have the flexibility we want without some flags to the method, or related methods as @bensigelman says.

More discussion in https://github.com/opentracing/opentracing-python/pull/13 but I think we must consider these 3 use cases as mentioned by @yurishkuro :

- eg. app server entry point -- start traces on incoming requests, or continue traces if context propagated
- eg. internal service entry point -- only continue traces if context propagated (no partial traces of this subsystem)
- eg. span within a process -- only continue traces from in-process parent span

My vote would be to have a `start_trace` which signals entry to a particular process etc, `create_span` which is used within a process.  This would cover eg. both the sampling and non-sampling cases.  Ideally, `start_trace` would take an optional parent span ID--in some languages, this would be augmented by `join_trace` as previously proposed, or perhaps the instrumenter can simply use `create_span` to affec this.